packages/expo-av/CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 - Add web support for recording. ([#8721](https://github.com/expo/expo/pull/8721) by [@WazzaJB](https://github.com/WazzaJB) and [@mnightingale](https://github.com/mnightingale))
 - Add permissions support for web. ([#8721](https://github.com/expo/expo/pull/8721) by [@mnightingale](https://github.com/mnightingale))
 - Add Audio `usePermissions` hook from modules factory. ([#13851](https://github.com/expo/expo/pull/13851) by [@bycedric](https://github.com/bycedric))
+- On iOS, add Audio metadata (title) updates for iOS via `setOnMetadataUpdate`. ([#14134](https://github.com/expo/expo/pull/14134) by [@dani-mp](https://github.com/dani-mp))
```

Comment:
I'm sorry I'm suggesting another update here 😓 
I've, for some reason, failed to remove the original `iOS` word position 😅 
```suggestion
- On iOS, add Audio metadata (title) updates via `setOnMetadataUpdate`. ([#14134](https://github.com/expo/expo/pull/14134) by [@dani-mp](https://github.com/dani-mp))
```

Reply:
It was probably my fault. I wanted to add the comma and missed the other change you proposed. 👍